.gitmodules

```diff
@@ -43,3 +43,7 @@
 [submodule "cmake/external/onnx-tensorrt"]
 	path = cmake/external/onnx-tensorrt
 	url = https://github.com/onnx/onnx-tensorrt.git
+[submodule "cmake/external/wil"]
```

Comment:
It'd probably be possible, but it would be rather awkward since we consume it as source as it's a header-only library. Keeping it as a submodule seems more natural and in line with convention (although if there's an existing precedent for this - let me know!)